floris/simulation/turbine.py

```diff
@@ -36,51 +36,6 @@
 from floris.utilities import cosd
 
 
-def _filter_convert(
```

Comment:
I should have made a better note about this than vaguely saying I removed it because it wasn't used. Ultimately, the solver is the only place where the `ix_filter` is use and it's always a Python list which is the one case where the `_filter_convert` function isn't needed.

I think I was going a bit heavy on deleting things when I removed this one, so I'm happy to add it back. The motivation to delete unused code is to have a less complex starting point for the v4 work.

Reply:
No, I think this function outlived its usefulness, so I'm fine with removing it. It made sense earlier on when we assumed others might be directly invoking some of the turbine calculations, but that's not the case.